README.md

```diff
@@ -130,22 +128,20 @@ trigger a run.
 **If I remove a configuration file, will kube-applier remove the associated Kubernetes object?**
 
 This is dependent on the value of `kube-applier.io/prune` (default `true`). If `true`,
-then a [default whitelist](https://github.com/utilitywarehouse/kube-applier/blob/master/kube/client.go#L35-L47) will be passed to the apply command.
+then kube applier will prune all namespaced resources.
 
-You can overwrite the whitelist with the annotation
-`kube-applier.io/prune-whitelist`. For instance:
+If you want kube applier to prune cluster resources, you can set
+`kube-applier.io/prune-cluster-resources` to `true`. Take care when using this
+feature as it will remove all cluster resources that have been created by
+kubectl and don't exist in the current namespace directory. Therefore, only use this feature if all
+of your cluster resources are defined under one namespace.
```

Comment:
"single managed directory"? 

Or something similar, technically its wrong to say "one namespace", since we are talking about cluster resources.